### DISCUSSION TITLE
Still need Support w/ a miniKP ?

### DISCUSSION THREAD

**Discussion by listenForward** [bangcorrupt/hacktribe#201] [General]
Hey BangCorrupt.
First of all, let me just say how exciting and inspiring your work w/ HackTribe is so far. I was just about to sell my e2s, and you've gotten me all smitten and curious w/ the unit.

Secondly, are you still looking for a KP-min ? does it have to be the "S" model ? I have one I could lend to you long term, especially if it will come back un-broken. If you need one to sacrifice/autopsy, I could send you one as a form of Support.

Cheers !

**Comment by bangcorrupt** [comment]
Hi, thank you for the generous offer, but I have a mini kp2s now.  It could be interesting to see if the 2 will run the 2s firmware (I think they have the same processor), but that's probably a job for someone else.  There is always a good chance things will be permanently destroyed when hacking them.  I'm slowly saving for a KingKorg, should only take a few years if I can find a cheap used one :smile:.

I like the ideas you posted for trigger mode and the UI stuff is similar to what I'm planning.  I want to make shortcuts work similar to Polyend Tracker, with the encoder functions modified depending on which button is held.
